jobs/build/coreos-installer_sync/extract.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -uo pipefail
```

Comment:
I will be skeptical to skip the `e` option. It could result in silent error without notice. Also I would like to see why the exit code is 1 on a successful extraction.

Reply:
if with `-e` it failed like https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/hack/job/ximhan/job/build%252Fcoreos-installer_sync/3/console

Reply:
+1 agree with @vfreex , let's not skip `e`.

Reply:
ok, updated in https://github.com/openshift-eng/aos-cd-jobs/pull/4070/commits/e1809ad7db1a521a0dd6f66bdfea5ebc480d517e